unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/Animations/VoiceChat/VoiceIconController.controller

```diff
@@ -0,0 +1,211 @@
+%YAML 1.1
```

Comment:
We should put the voice chat animations in a folder containing all the `AvatarName` related stuff too, instead of the avatar `Animations` folder. This way all the assets related with the name UI are in the same place.